config/job_conf.xml

```diff
@@ -1,495 +1,547 @@
 <?xml version="1.0"?>
 <!-- A sample job config for sketching the k8s use case with shared fs -->
 <job_conf>
-  <plugins>
-    <plugin id="local" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner" workers="4"/>
-    <plugin id="k8s" type="runner" load="galaxy.jobs.runners.kubernetes:KubernetesJobRunner">
-      <param id="k8s_use_service_account">true</param>
-      <param id="k8s_persistent_volume_claim_name">galaxy-pvc</param>
-      <!-- The following mount path needs to be the initial part of the "file_path" and "new_file_path" paths
-            set in universe_wsgi.ini (or equivalent general galaxy config file).  -->
-      <param id="k8s_persistent_volume_claim_mount_path">/opt/galaxy_data</param>
-      <param id="k8s_namespace">default</param>
-      <param id="k8s_supplemental_group_id">0</param>
-      <param id="k8s_fs_group_id">0</param>
-      <param id="k8s_pull_policy">IfNotPresent</param>
-      <!-- Allows pods to retry up to this number of times, before marking the Job as failed -->
-      <param id="k8s_pod_retrials">1</param>
-    </plugin>
-  </plugins>
-  <handlers>
-    <handler id="main"/>
-  </handlers>
-  <destinations default="local">
-    <destination id="local" runner="local"/>
-    <destination id="dynamic-k8s" runner="dynamic">
-      <param id="type">python</param>
-      <param id="function">k8s_wrapper</param>
-      <param id="docker_enabled">true</param>
-    </destination>
-    <destination id="dynamic-k8s-tiny" runner="dynamic">
-      <param id="type">python</param>
-      <param id="function">k8s_wrapper_tiny</param>
-      <resubmit condition="memory_limit_reached" destination="dynamic-k8s-small"/>
-    </destination>
-    <destination id="dynamic-k8s-small" runner="dynamic">
-      <param id="type">python</param>
-      <param id="function">k8s_wrapper_small</param>
-      <resubmit condition="memory_limit_reached" destination="dynamic-k8s-medium"/>
-    </destination>
-    <destination id="dynamic-k8s-medium" runner="dynamic">
-      <param id="type">python</param>
-      <param id="function">k8s_wrapper_medium</param>
-      <resubmit condition="memory_limit_reached" destination="dynamic-k8s-large"/>
-    </destination>
-    <destination id="dynamic-k8s-large" runner="dynamic">
-      <param id="type">python</param>
-      <param id="function">k8s_wrapper_large</param>
-      <resubmit condition="memory_limit_reached" destination="dynamic-k8s-xlarge"/>
-    </destination>
-    <destination id="dynamic-k8s-xlarge" runner="dynamic">
-      <param id="type">python</param>
-      <param id="function">k8s_wrapper_xlarge</param>
-    </destination>
+    <plugins>
```

Comment:
There is something wrong with this commit, this file should have such a large diff